review/reviewer/speed.md

```diff
@@ -56,17 +56,16 @@ returning from a meeting, coming back from the breakroom, etc.
 ## Fast Responses {#responses}
 
 When we talk about the speed of code reviews, it is the _response_ time that we
-are concerned with, as opposed to how long it takes a CL to get through the
-whole review and be submitted. The whole process should also be fast, ideally,
-but **it's even more important for the _individual responses_ to come quickly
```

Comment:
This sentence got shortened a bit and for me it's not clear now with the modification. Was this intentional or accidental?

Reply:
fixed